src/null-safety/index.md

```diff
@@ -334,6 +334,18 @@ If you find any issues with null safety please [give us feedback.][]
 
 [language version]: /guides/language/evolution#language-versioning
 
+## Known issues
+
+Not all parts of the Dart SDK support null safety yet,
+as some parts still need additional work to
+[migrate to null safety](https://dart.dev/null-safety/migration-guide).
+
+We're currently aware of the following issues:
+
+  * Migration of the pub.dev packages owned by the Dart team
```

Comment:
I wouldn't call these package part of the SDK. Should we broaden the intro?

Reply:
Sure. Dart platform? Or just Dart?

Reply:
I made a couple of suggestions above that might help